### PR TITLE
Bug 1914125: updated localVolume CRD and CSV files examples to encourage using /dev/disk/by-id...

### PR DESCRIPTION
**Pull request by sp98**
Signed-off-by: Santosh Pillai <sapillai@redhat.com>

### CODE REVIEW

File: manifests/4.7/local-volumes.crd.yaml [hunk 1]
```diff
@@ -64,7 +64,9 @@ spec:
                         type: string
                       devicePaths:
                         description: 'A list of devices which would be chosen for local storage.
-                        For example - ["/dev/sda", "/dev/sdb", "/dev/disk/by-id/ata-crucial"]'
+                        For example - ["/dev/sda", "/dev/sdb", "/dev/disk/by-id/ata-crucial"]
+                        Note: Use the unique /dev/disk/by-id/.. name, if available, to avoid issues
```
Comment (huffmanca) [inline manifests/4.7/local-volumes.crd.yaml:68]: Really just a nitpick, but I don't think we should have a Note in the CRD. Perhaps something similar to:

> It is strongly recommended to use the unique /dev/disk/by-id/... name if available, as these identifiers persist across node restarts.

File: opm-bundle/manifests/local-volumes.crd.yaml [hunk 2]
```diff
@@ -64,7 +64,9 @@ spec:
                         type: string
                       devicePaths:
                         description: 'A list of devices which would be chosen for local storage.
-                        For example - ["/dev/sda", "/dev/sdb", "/dev/disk/by-id/ata-crucial"]'
+                        For example - ["/dev/sda", "/dev/sdb", "/dev/disk/by-id/ata-crucial"]
+                        Note: Use the unique /dev/disk/by-id/.. name, if available, to avoid issues
```
Comment (huffmanca) [inline opm-bundle/manifests/local-volumes.crd.yaml:68]: Same comment as above.

File: manifests/4.7/local-volumes.crd.yaml (hunk 1 above)
Comment (sp98) [reply manifests/4.7/local-volumes.crd.yaml]: updated.

File: opm-bundle/manifests/local-volumes.crd.yaml (hunk 2 above)
Comment (sp98) [reply opm-bundle/manifests/local-volumes.crd.yaml]: updated.

**Pull request merged**